src/utils.js

```diff
@@ -85,6 +85,41 @@ exports.flattenStartCase = (object) => {
 };
 
 /**
+ * @summary Remove paths contained in strings
+ * @function
+ * @public
+ *
+ * @param {String} str - string potentially containing paths
+ * @returns {String} string with basenamed paths if any
+ *
+ * @example
+ */
+exports.removeContainedPaths = (str) => {
+  const absolutify = (pathname) => {
```

Comment:
`absolutify` is the wrong name here isn't it? Should it be `relativify` or `make_relative`?